### PR TITLE
Food

### PR DESCRIPTION
**Pull request by AlvaroH2001**
## Description
For this pull request you will see quite a bit of file changes and additions. To summarize, I edited the food page to include the new component MenuTabs, this component holds the three tabs, Order Food, Past Orders, and Meal Customization, each of course also being their own component. Order Food highlights a breakfast, lunch, and dinner menu with relevant nutritional information and a cool animation for people to select things to add to their order. The PastOrders component shows a list of past orders from the last 7 days to help keep track of nutrition. And of course MealCustomization allows for the inclusion of dietary restrictions and allergies. JSON data for these menu items and relevant info was added to the data folder. The components were added to the components folder. Pictures of the food were also added to our images. 
## Testing
Due to the size of this PR I figured it would be best to separate the testing into a future PR.
## Possible Errors
The possibility of odd formatting if the webpage is shrunken down or stretched. If you are reviewing this PR please look through and point out any clear mistakes I may have made or best practices that I may have missed, I would like this page to essentially be _complete_.
## Images
Screenshot 2024-04-16 at 8 31 44 PM
Screenshot 2024-04-16 at 8 31 53 PM
Screenshot 2024-04-16 at 8 32 12 PM
Screenshot 2024-04-16 at 8 32 23 PM
Screenshot 2024-04-16 at 8 32 31 PM

### CODE REVIEW

File: small-talk/public/img/food/breakfast/cereal-milk.jpg
Comment (blake-gilmore) [inline small-talk/public/img/food/breakfast/cereal-milk.jpg:1]: Yummy

**Review by blake-gilmore — APPROVED** [review]
Really awesome work. This looks fantastic!

File: small-talk/app/components/foodPage/MenuTabs.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+                    {mainTab === 'mealCustomization' && (
+                        // Render meal customization component for "Meal Customization" tab
+                        <div className="p-4">
+                            <MealCustomization />
+                        </div>
+                    )}
+                </div>
+            </div>
+            </div>
```
Comment (blake-gilmore) [inline small-talk/app/components/foodPage/MenuTabs.js:103]: Great use of conditional renders for these components.

**Review by Redblaze74 — APPROVED** [review]
Great work, the layout of the page and attention to details is amazing. The only nitpick from me would be just swapping from CSS files to Tailwind CSS but that's really it.

**Comment by aiharc1** [comment]
Wow nice job Alvaro. The addition of the food page is huge and looks GREAT. I really want to approve this but the only reason why I cant is because of the failure from the build. Other than that super nice work!

**Review by junxjr — APPROVED** [review]
This is really nice. I love the food page! I looked at the error for not passing the checks. It looks like you just have to create at least one test case for food page. I also think it should be separated into multiple PRs for the test cases since the file is big, but maybe try to just have a single test case to pass the check for now.

**Review by jusasin — APPROVED** [review]
This is great work! Wonderful layout to the food page and the set up is very simple and easy to follow for anyone of any age to follow.

**Pull request merged**